### DISCUSSION TITLE
2023년 06월 30일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#63] [General]
# 2023년 06월 30일

LOG 베로위키 다시 영업합니다 구경 자주 오세요~ 배운 것 JPA ddl-auto는 create-drop이 default 값이다. 궁금한 것 느낀 것

https://vero.wiki/2023-06-30/

**Comment by Choi-JJunho** [comment]
issue 삭제 가능합니다~

**Reply by Cyma-s** [reply]
가짜 뉴스 들켰네 ㅋㅋ
